website/docs/components/alert/partials/code/how-to-use.md

```diff
@@ -43,6 +43,29 @@ Optionally, you can pass only `title` or only `description`.
 </Hds::Alert>
 ```
 
+### Tag
+
+The `tag` value for the Alert Title changes the HTML element wrapping the title content. When organizing the content on a webpage, the heading levels should reflect the structure of the page. For example, if an Alert appears directly below the main heading of the page, it should be `"h2"`.
+
+```handlebars
+<div class="doc-alert-flex-layout">
+ <Hds::Text::Display @tag="h1" @size="500">Edit credit card</Hds::Text::Display>
+</div>
+<Hds::Alert @type="inline" @color="critical" as |A|>
+  <A.Title @tag="h2">Form submission error</A.Title>
+  <A.Description>Correct the formatting of the following fields to update your user profile:</A.Description>
+  <A.Description>
+    <Hds::Link::Inline @href="#" @color="secondary">Expiration date</Hds::Link::Inline>
+  </A.Description>
+</Hds::Alert>
+```
+
+!!! Insight
+
+The default `tag` is `"div"` because the correct value is dependent on the individual page. We highly encourage consumers to update the `tag` because it makes what screen readers announce to users match the visual experience.
```

Comment:
Suggestion to update this text with the same text in the beginning of this review. It references a specific SC and is is a little more emphatic about the need for the change.